### PR TITLE
[Xcodeproj] Add “SKIP_INSTALL = YES” to framework target

### PR DESCRIPTION
**Pull request by norio-nomura**
Fix https://bugs.swift.org/browse/SR-3375

### CODE REVIEW

File: Sources/Xcodeproj/pbxproj().swift [hunk 1]
```diff
@@ -351,6 +351,7 @@ func xcodeProject(
                 targetSettings.common.PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)"
                 targetSettings.common.PRODUCT_MODULE_NAME = "$(TARGET_NAME:c99extidentifier)"
                 targetSettings.common.PRODUCT_BUNDLE_IDENTIFIER = module.c99name.mangledToBundleIdentifier()
+                targetSettings.common.SKIP_INSTALL = "YES"
```
Comment (aciidb0mb3r) [inline Sources/Xcodeproj/pbxproj().swift]: We don't need to do this for executables?

**Comment by aciidb0mb3r** [comment]
Thank you so much!
LGTM, @abertelrud can you also take a quick look.

File: Sources/Xcodeproj/pbxproj().swift (hunk 1 above)
Comment (norio-nomura) [inline Sources/Xcodeproj/pbxproj().swift]: `SKIP_INSTALL` is not configured on executable target in Xcode's default project template.

**Comment by aciidb0mb3r** [comment]
@swift-ci please test

**Comment by abertelrud** [comment]
Looks good to me too.  In general we should try to match what the templates produce when there isn't anything more specific that can be inferred from the package.

**Comment by norio-nomura** [comment]
Thanks! 🙏

**Pull request merged**